rllib/algorithms/dreamer/dreamer.py

```diff
@@ -201,52 +201,31 @@ def _postprocess_gif(gif: np.ndarray):
     return frames
 
 
-class EpisodicBuffer(object):
-    def __init__(self, max_length: int = 1000, length: int = 50):
-        """Stores episodes and samples chunks of size ``length`` from episodes.
+class EpisodeSequenceBuffer(ReplayBuffer):
```

Comment:
Renamed it, to highlight that it's not a normal ReplayBuffer that replays episodes. Also, it's a little different than a only setting storage_unit=="sequences", since we would only be able to sample non-overlapping sequences.

Reply:
Makes sense.